admin/admin.go

```diff
@@ -81,5 +121,26 @@ func setupRouter() *gin.Engine {
 	router.GET("/ready", func(c *gin.Context) {
 		c.String(http.StatusOK, "OK")
 	})
+
 	return router
 }
+
+func (m *RPCMethods) GetConfig(args *struct{}) *config.NetworkConfig {
+
+	m.Log.Info("GetConfig called")
+	return m.NetworkConfig
+}
+
+func (m *RPCMethods) GetL1Addresses(args *uint64) *map[string]string {
+	chain := filterByChainID(m.NetworkConfig.L2Configs, *args)
+	if chain == nil {
+		m.Log.Info("chain not found")
+		return nil
+	}
+	reply := map[string]string{
+		"L2CrossDomainMessenger": chain.L2Config.L1Addresses.L1CrossDomainMessengerProxy.String(),
+		"L2StandardBridge":       chain.L2Config.L1Addresses.L1StandardBridgeProxy.String(),
```

Comment:
these are named wrong - they should be named identically to the ones they refer to
L1CrossDomainMessengerProxy
L1StandardBridgeProxy

we should also add all of these?

```
	AddressManager                    Address `json:"AddressManager" toml:"AddressManager"`
	L1CrossDomainMessengerProxy       Address `json:"L1CrossDomainMessengerProxy" toml:"L1CrossDomainMessengerProxy"`
	L1ERC721BridgeProxy               Address `json:"L1ERC721BridgeProxy" toml:"L1ERC721BridgeProxy"`
	L1StandardBridgeProxy             Address `json:"L1StandardBridgeProxy" toml:"L1StandardBridgeProxy"`
	L2OutputOracleProxy               Address `json:"L2OutputOracleProxy" toml:"L2OutputOracleProxy,omitempty"`
	OptimismMintableERC20FactoryProxy Address `json:"OptimismMintableERC20FactoryProxy" toml:"OptimismMintableERC20FactoryProxy"`
	OptimismPortalProxy               Address `json:"OptimismPortalProxy,omitempty" toml:"OptimismPortalProxy,omitempty"`
	SystemConfigProxy                 Address `json:"SystemConfigProxy" toml:"SystemConfigProxy"`
	ProxyAdmin                        Address `json:"ProxyAdmin" toml:"ProxyAdmin"`
	SuperchainConfig                  Address `json:"SuperchainConfig,omitempty" toml:"SuperchainConfig,omitempty"`

```

Reply:
done✅